libqtile/widget/thermal_zone.py

```diff
@@ -0,0 +1,37 @@
+from libqtile.widget import base
+from libqtile.log_utils import logger
+
+
+class ThermalZone(base.ThreadPoolText):
+    orientations = base.ORIENTATION_HORIZONTAL
+    defaults = [
+        ('update_interval', 2.0, 'Update interval'),
+        ('zone', '/sys/class/thermal/thermal_zone0/temp', 'Thermal zone'),
+        ('format', '{temp}°C', 'Thermal zone display format'),
+        ('fgcolor_crit', 'ff0000', 'Font color on critical values'),
+        ('fgcolor_high', 'ffaa00', 'Font color on high values'),
+        ('fgcolor_normal', 'ffffff', 'Font color on normal values'),
+        ('crit', 70, 'Critical temperature level'),
+        ('high', 50, 'High themperature level'),
+    ]
+
+    def __init__(self, **config):
+        super().__init__("", **config)
+        self.add_defaults(ThermalZone.defaults)
+
+    def poll(self):
+        try:
+            with open(self.zone) as f:
+                value = round(int(f.read().rstrip()) / 1000)
+                if value < self.high:
+                    self.layout.colour = self.fgcolor_normal
+                elif value in range(self.high, self.crit):
```

Comment:
A bit simpler perhaps:
```suggestion
                elif value < crit:
```

Reply:
Done